### DISCUSSION TITLE
is there any possibility to create local proxy server for apis like gemini,groq? because right now i see that option for local models only

### DISCUSSION THREAD

**Discussion by Greatz08** [BerriAI/litellm#2288] [General]
i am asking for this is because i wanted to connect groq with flowise and currently there was no option to integrate so i thought if i create local proxy server and use local ai functionality of it to connect with groq and provide api key and chat using groq so thought to ask.

**Comment by ghost** [comment]
yea - https://docs.litellm.ai/docs/providers

all of those are supported. anything missing in docs?

**Comment by Greatz08** [comment]
i resolved this issue in discord closing now :-))
